core-bundle/contao/templates/twig/content_element/swiper.html.twig

```diff
@@ -4,35 +4,41 @@
 {#
 Global settings for all sliders
 - Create a new template 'swiper.html.twig' (content_element/swiper.html.twig)
-- Copy the block ‘init_options’ and customise them (https://swiperjs.com/)
+- Copy the block ‘init_options’ and adjust them (https://swiperjs.com/)
 
 Custom template
 - Create folder 'swiper' if it does not exist
 - Create a new template 'custom.html.twig' in folder 'swiper' (content_element/swiper/custom.html.twig)
-- Copy the section ‘Slider settings’ and customise them (https://swiperjs.com/)
-- After the section ‘Slider settings’, assign the new setting to ‘slider_attributes’:
-
-{% set slider_attributes = attrs().set('data-settings', slider_settings|json_encode) %}
-
+- Copy the section ‘Custom slider settings’ and adjust them (https://swiperjs.com/)
 #}
 
-{# Slider settings #}
+
+{# Custom slider settings #}
+{#
 
 {% set slider_settings = {
-    speed: speed,
-    initialSlide: offset,
-    loop: loop,
+    effect: 'fade',
+    fadeEffect: {
+        crossFade: true
+    },
+    pagination: {
+        el: '.swiper-pagination',
+        type: 'progressbar',
+    },
 } %}
-{% if delay > 0 %}
-    {% set slider_settings = slider_settings|merge({ autoplay: {delay: delay} }) %} 
-{% endif %}
 
-{# End Slider settings #}
+{% set slider_attributes = attrs().set('data-settings', slider_settings|json_encode) %}
+
+#}
+{# End custom slider settings #}
 
 {% block content %}
     {% set slider_attributes = attrs()
         .addClass('swiper')
-        .set('data-settings', slider_settings|json_encode )
+        .set('data-delay', delay)
+        .set('data-speed', speed)
+        .set('data-offset', offset)
+        .set('data-loop', loop)
```

Comment:
The `data-settings` attribute is now missing.

Reply:
The data-settings attribute is set via the custom template.

We do not have any additional settings in the swiper-template, as an empty attribute would always be transmitted

Reply:
I am not sure what you mean. This is how it should work:

* The controller build the Swiper options and passes it to the template as an array.
* The template then adds it as the `data-settings` attribute JSON encoded.
* The Swiper init script uses that attribute to initialize the Swiper instances.

i.e.:

1. Revert d8d36d107706b26bf435fac75911eb066b5cab44
2. Apply https://github.com/contao/contao/pull/7470#discussion_r1727350306
3. Change the `SwiperController` to pass the swiper settings array to the template.

Reply:
See https://github.com/delirius/contao/pull/1